bedrock/firefox/templates/firefox/welcome/page14.html

```diff
@@ -0,0 +1,98 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ ftl('welcome-page14-page-title-alt') }}{% endblock %}
+
+{% block page_image %}{{ static('img/products/vpn/common/social-share.png') }}{% endblock %}
+
+{% block body_class %}welcome-page14{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page10') }}
+  {{ css_bundle('firefox_welcome_page14') }}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle("firefox_welcome_page14") }}
+{% endblock %}
+
+{% set show_coupon = switch('vpn-moments-coupon-code') %}
+{% set coupon_code = 'VPNPRIVACY20' %}
+
+{% block shoulder_cta %}{% endblock %}
+
+
+
+{% block content_intro %}
+  <section class="c-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <img class="c-main-image" src="{{ static('protocol/img/logos/mozilla/vpn/logo-flat.svg') }}" width="60" alt="">
+      <h1 class="c-main-title"> {% if show_coupon %}{{ ftl('welcome-page14-page-title') }}{% else %}{{ ftl('welcome-page14-page-title-alt') }}{% endif %}</h1>
+      <p class="c-main-tagline">{{ ftl('welcome-page14-main-description') }}</p>
+      <div class="c-main-cta">
+        {{ vpn_product_referral_link(
+          referral_id='welcome-page14',
+          link_text=ftl('welcome-page14-get-mozilla-vpn'),
+          class_name='mzp-t-product mzp-t-xl',
```

Comment:
Suggestion (non-blocking): you can add `page_anchor='#pricing',` here if you wanted to link directly to the pricing section on the landing page. Not sure if this is a requirement or not, just calling it out.